Documentation/scaling.rst

```diff
@@ -0,0 +1,88 @@
+Scaling
+=======
+
+When working with SI units in earthquake scenarios numbers might get very large.
+Rescaling the equations might be advantageous, e.g. when working with single precision arithmetic.
+In this section, we show how to properly scale the elastic wave equation.
+
+The elastic wave equation in velocity-stress form with source terms is given by
+
+.. math::
+
+   \begin{aligned}
+     \frac{\partial\sigma_{ij}}{\partial t}
+           - \lambda \delta_{ij}\frac{\partial u_k}{\partial x_k}
+           - \mu \left(
+               \frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i}
+            \right) &= s_i \\
+     \rho\frac{\partial u_i}{\partial t} - \frac{\partial \sigma_{ij}}{\partial x_j} &= f_i
+   \end{aligned}
+
+We define scaled quantities
+
+.. math::
+
+   \bar{x}_i = \frac{x_i}{L}, \quad \bar{u}_i = \frac{u_i}{u_c}, \quad
+       \bar{\sigma}_{ij} = \frac{\sigma_{ij}}{\sigma_c}
+
+Inserting these into the elastic wave equation gives
+
+.. math::
+
+   \begin{aligned}
+       \sigma_c\frac{\partial \bar{\sigma}_{ij}}{\partial t}
+         - u_cL^{-1} \lambda \delta_{ij}\frac{\partial \bar{u}_k}{\partial\bar{x}_k}
+         - u_c L^{-1} \mu \left(
+            \frac{\partial \bar{u}_i}{\partial\bar{x}_j} +
+            \frac{\partial \bar{u}_j}{\partial\bar{x}_i}\right) &= s_i \\
+       \rho u_c\frac{\partial \bar{u}_i}{\partial t} - L^{-1}\sigma_c
+         \frac{\partial \bar{\sigma}_{ij}}{\partial \bar{x}_j} &= f_i
+   \end{aligned}
+
+Multiplying the first equation with :math:`\sigma_c^{-1}`, the second with :math:`L\sigma_c^{-1}`,
+and defining
```

Comment:
Why not dividing my sigma_c?

Reply:
I guess that's a matter of taste :-). The second equation has multiplication and division thus I found it easier to always write "multiplication".

Reply:
Right, missed the L term.